tests/src/Test/Tasty/Vivado/GenTcl.hs

```diff
@@ -0,0 +1,151 @@
+{-# LANGUAGE QuasiQuotes #-}
+{-# LANGUAGE RecordWildCards #-}
+
+-- | Generate a TCL script to simulate generated VHDL
+--
+-- Run with @vivado -mode batch -source ...@
+module Test.Tasty.Vivado.GenTcl ( HdlSource (..), tclFromManifest ) where
+
+import Paths_clash_testsuite (getDataDir)
+
+import Clash.Driver.Manifest (Manifest (..))
+import Test.Tasty.Common (getManifests)
+
+import Data.Char (isLower)
+import Data.List (isSuffixOf)
+import Data.String.Interpolate (i)
+
+import System.FilePath ((</>))
+import qualified Data.Text as T
```

Comment:
Our style guide would like us to order them differently:
```suggestion
import Data.Char (isLower)
import Data.List (isSuffixOf)
import qualified Data.Text as T
import Data.String.Interpolate (i)
import System.FilePath ((</>))

import Clash.Driver.Manifest (Manifest (..))

import Paths_clash_testsuite (getDataDir)
import Test.Tasty.Common (getManifests)
```

Reply:
Oh you're right, sorry :grimacing: